presto-function-namespace-managers/src/main/java/com/facebook/presto/functionNamespace/SqlInvokedFunctionNamespaceManagerConfig.java

```diff
@@ -14,25 +14,28 @@
 package com.facebook.presto.functionNamespace;
 
 import com.facebook.airlift.configuration.Config;
+import com.facebook.airlift.json.JsonCodec;
+import com.facebook.presto.spi.function.FunctionImplementationType;
 import com.facebook.presto.spi.function.RoutineCharacteristics;
-import com.google.common.base.Splitter;
-import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.ImmutableMap;
 import io.airlift.units.Duration;
 import io.airlift.units.MinDuration;
 
-import java.util.Set;
+import java.util.Map;
 
+import static com.facebook.airlift.json.JsonCodec.mapJsonCodec;
+import static com.facebook.presto.spi.function.FunctionImplementationType.BUILTIN;
 import static com.facebook.presto.spi.function.RoutineCharacteristics.Language.SQL;
 import static java.util.concurrent.TimeUnit.HOURS;
 import static java.util.concurrent.TimeUnit.MINUTES;
 
 public class SqlInvokedFunctionNamespaceManagerConfig
 {
-    private static final Splitter LANGUAGE_SPLITTER = Splitter.on(',').omitEmptyStrings().trimResults();
+    private static final JsonCodec<Map<RoutineCharacteristics.Language, FunctionImplementationType>> FUNCTION_LANGUAGES_CODEC = mapJsonCodec(RoutineCharacteristics.Language.class, FunctionImplementationType.class);
```

Comment:
Can we static import? You can ask IntelliJ to do this for you by pressing `option + return` and select the 1st option "Add on-demand static import" ...

Reply:
That would import ".*" for me. How to avoid that?

Reply:
You can use `ctrl + option + o` on this file to optimize import.